r-package/grf/R/average_treatment_effect.R

```diff
@@ -18,6 +18,7 @@
 #'
 #' @param forest The trained forest.
 #' @param target.sample Which sample to aggregate treatment effects over.
+#' @param subset The subset to calculate CATE for.
```

Comment:
It's important to emphasize that subsetting should only be done based on subgroups defined by the X-variable (as opposed to the W- or Y-variables).

Also, please add an example using subsetting (e.g., at the end of the examples section, add one that subsets on X[,1] < 0).